Vagrant-Up.md

```diff
@@ -0,0 +1,45 @@
+The following instruction will help you install the necessary software and also how to set up your first Vagrant and Virtual Box. This will create a working environment so you can test your web site on your local machine. The <b>Vagrantfile</b> can be configured to your specific environment so that anyone working with you can make and see changes no matter if they use Linux, Mac OS X, or Windows. Using this virtual environment, you will be able to see your PHP scripts run, build and work with your databases, and so much more. So, without further or do, let's get down to business. 
+
+
+Check out the following links and install each program. Be sure that it matches your OS (Operating System).
+
+https://www.virtualbox.org/
+
+https://www.vagrantup.com/downloads.html
+
+https://git-scm.com/downloads
+
+Now that we have the necessary programs installed, let’s get down to business.
+
+<b>Git</b> has several different programs we can use here. Let’s open the <b>Git GUI</b>. We also can start up <b>Virtual Box</b> so that we can see the box running.
+
+Once you run the <b>Git GUI</b>, you’ll see several options. Let’s select the top option, </b>“Create New Repository”</b>.
+
+Select the <b>“Browse”</b> button and select the drive you want to install and run your virtual machine / server from. You can right click in the folder area and create a new folder. Let’s name it <b>FreeCodeCampMachine</b>.
+
+Now, you’ll see the <b>Git Gui</b> and beside Directory, you should see <b>C:/FreeCodeCampMachine</b>.
```

Comment:
I think your tutorial is based on Windows environment only. If so, provide proper headers of the subsection. Also I would encourage you to add Linux + Mac specific instructions too. If you dont know thats fine, just leave the appropriate subsection headers.